common/src/main/java/com/graphicsfuzz/common/transformreduce/GlslShaderJob.java

```diff
@@ -43,6 +46,7 @@
 import java.util.stream.Collectors;
 import org.apache.commons.lang3.tuple.ImmutablePair;
 
+
```

Comment:
Avoid adding whitespace if there's not a reason to do so.

Reply:
done